subscribers:
  
  queue_size: 5

  image_rgb:
    topic: /butia_vision/bvb/image_rgb_raw

  image_depth:
    topic: /butia_vision/bvb/image_depth_raw

  camera_info:
    topic: /butia_vision/bvb/camera_info

server:

  image_request:
    service: /butia_vision/is/image_request

parameters:
  buffer_size: 150
  use_exact_time: true